app/src/main/java/com/venuenext/venuenextsdkdemo/MainActivity.kt

```diff
@@ -56,6 +57,34 @@ class MainActivity : AppCompatActivity(), BottomNavigationView.OnNavigationItemS
         // Setting a reselected listener will disallow selection of the same tab
         // bottomNavigationView.setOnNavigationItemReselectedListener { /* Disallow reselection */ }
 
+        // This is an example of how to initialize the SDK if you have reveived a JSON config file
```

Comment:
Lets just have this be the example

Reply:
@chriscioli-vn thought we shouldn’t have this be the example yet because nobody has a config file yet. 

I’m open to either.

Reply:
We can un-comment it. If anyone runs the app, they'll have to replace the config file with their own anyway. I still think we should have both methods of initialization in the demo app until we are no longer using the "old" method.